launch/config.yml

```
@@ -1,5 +1,5 @@
 relaychain:
-  image: parity/polkadot:v0.9.3
+  image: parity/polkadot:v0.9.4
```

v0.9.4 is not released yet
```suggestion
  image: parity/polkadot:v0.9.3
```